src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -142,9 +142,9 @@ public void Initialize(TestLoggerEvents events, string testRunDirectory)
             }
 
             // Register for the events.
-            events.TestRunMessage += this.TestMessageHandler;
```

Comment:
We have been following this convention to use 'this' with members, makes it readable, any specific reason why we are removing it ?